packages/bpk-docs/src/template.js

```diff
@@ -39,6 +39,14 @@ export default ({ head = {}, html = '', assets = {} }) => `<!doctype html>
   ${html}
 </div>
 
+<script type="text/javascript">
+  window.BPK_DOC_GLOBALS = window.BPK_DOC_GLOBALS || {};
+  window.BPK_DOC_GLOBALS.entries = {
+    docs: '${assets.docs.js}',
```

Comment:
Why is docs there? we don't use that do we?

Reply:
Just adding all entries there, not really required but I don't see any harm in having it there.

Reply:
I'd prefer to remove it, but don't block this PR on me.